include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -2358,11 +2376,13 @@ template <int n_components_, int dim, int spacedim, typename Number>
 FEPointEvaluation<n_components_, dim, spacedim, Number>::FEPointEvaluation(
   const NonMatching::MappingInfo<dim, spacedim, Number> &mapping_info,
   const FiniteElement<dim, spacedim>                    &fe,
-  const unsigned int first_selected_component)
+  const unsigned int first_selected_component,
+  const bool         lexicographic_numbering)
   : FEPointEvaluationBase<n_components_, dim, spacedim, Number>(
       mapping_info,
       fe,
       first_selected_component)
+  , lexicographic_numbering(lexicographic_numbering || this->renumber.empty())
```

Comment:
Maybe you could add a short explanation what `renumber` refers to (you had this in one of the comment to a previous PR/issue, I think that could help).

Reply:
Do you want me to make a direct comment here? The variable is documented in the class definition. (We could improve the documentation there.)

Reply:
Here I had the impression that it might help to understand the reader of the code why the second check is necessary. But after thinking again, I think it is clear that we have to force lexicographic numbering if no other numbering has been stored in the class. But I agree with your suggestion, it could make sense to somewhat extend the documentation of the member variable.

Reply:
That's the place where we check the FEs DoF numbering for lexicographic:
https://github.com/dealii/dealii/blob/5a760040edb2086d624fb6f3ad38362d1a588fa4/include/deal.II/matrix_free/fe_point_evaluation.h#L1963-L1969